.vscode/settings.json

```diff
@@ -64,4 +64,7 @@
     "github-actions.remote-name": "upstream",
     // NOTE: This disable the plugins formatting so astyle is used.
     "C_Cpp.formatting": "disabled",
+    "files.associations": {
+        "cmath": "cpp"
```

Comment:
Is this change intended?

Reply:
Ah, when did that slip in? No, def not intended. Will remove once I'm home

Reply:
Should be good now @Maleclypse